lib/database/friend.php

```diff
@@ -158,6 +160,7 @@ function getAllFriendsProgress($user, $gameID, &$friendScoresOut): int
                     FROM Achievements AS ach
                     WHERE ach.GameID = '$gameID' AND ach.Flags = 3
                 ) AS Inner1 ON Inner1.ID = aw.AchievementID
+                WHERE aw.HardcoreMode = " . AchievementAwardType::Softcore . "
```

Comment:
This is specific to the Friends Ranking on the user page and not the Global Ranking filtered by friends. For some reason I felt that it could include all unlocks as more of a tracking method for friends but it's probably best to change this to hardcore to prevent confusion and inconsistency. Still pending a fix here I'll look further into it